migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -14,10 +14,11 @@ You can migrate data to a different storage class, for example, from Red Hat Glu
 
 Optionally, you can use the xref:migration-understanding-cpma_{context}[Control Plane Migration Assistant (CPMA)] to assist you in migrating control plane settings.
 
-[IMPORTANT]
-====
-Before you begin your migration, be sure to review the information on xref:../../migration/migrating-3-4/planning-migration-3-to-4.adoc#planning-migration-3-to-4[planning your migration].
-====
+// TODO: Add back in once 4.3 planning content has been added
+// [IMPORTANT]
+// ====
+// Before you begin your migration, be sure to review the information on xref:../../migration/migrating-3-4/planning-migration-3-to-4.adoc#planning-migration-3-to-4[planning your migration].
+// ====
```

Comment:
You can comment out the whole section with `////` before and after the text. Not worth changing here.

Reply:
Yeah I did, though it's usually faster for me to select all and ctrl+/ to comment out all lines. Though thanks for sharing just in case. I probably should do that just so it's a cleaner diff.